module-services/service-cellular/CellularUrcHandler.cpp

```diff
@@ -78,6 +78,9 @@ void CellularUrcHandler::Handle(Cusd &urc)
     if (!message) {
         return;
     }
+    auto logMessage = message->substr(0, 16);
```

Comment:
Maybe something like this?
`constexpr size_t giveItaName = 16; message->substr(0, giveItaName);`

Reply:
done